packages/core/src/node/initRsbuild.ts

```diff
@@ -244,16 +242,72 @@ async function createInternalBuildConfig(
           .rule('css-virtual-module')
           .test(/\.rspress[\\/]runtime[\\/]virtual-global-styles/)
           .merge({ sideEffects: true });
+
+        if (isServer) {
+          chain.output.filename('main.cjs');
+          chain.optimization.minimize(false);
```

Comment:
we can use Rsbuild's config instead of using the chain, such as `output.minify` or `output.filename`

Reply:
<img width="200" alt="image" src="https://github.com/web-infra-dev/rspress/assets/79413249/58c9184f-3f83-40c9-aea1-2b208ed70790">

```typescript
// rsbuild
filename: {
  js: 'main.cjs',
},
```

I tried to set it this way, but the result was `[name].js`

Reply:
<img width="400" alt="image" src="https://github.com/web-infra-dev/rspress/assets/79413249/d8cd4e9e-2100-4b07-94d6-cd2bca370dd7">

Reply:
open a issue https://github.com/web-infra-dev/rsbuild/issues/2787